.appveyor.yml

```diff
@@ -0,0 +1,13 @@
+image: Ubuntu
```

Comment:
Tercer fichero que me encuentro para exactamente el mismo sistema de CI y con prácticamene las mismas órdenes. CircleCI no es malo de por si, es malo porque al haberlo usado ya n personas, la persona n+1 no tenía más que copiarlo. Se trata de que aprendáis y entendáis lo que estás haciendo, no de que pongáis n sitios de integración continua.